cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/CreateController.java

```diff
@@ -0,0 +1,108 @@
+package cloud.foundry.cli.services;
+
+import cloud.foundry.cli.crosscutting.beans.ServiceBean;
+import cloud.foundry.cli.crosscutting.beans.SpaceDevelopersBean;
+import cloud.foundry.cli.crosscutting.exceptions.CreationException;
+import cloud.foundry.cli.crosscutting.mapping.CfOperationsCreator;
+import cloud.foundry.cli.crosscutting.util.FileUtils;
+import cloud.foundry.cli.crosscutting.util.YamlCreator;
+import cloud.foundry.cli.operations.ServicesOperations;
+import cloud.foundry.cli.operations.SpaceDevelopersOperations;
+
+import org.cloudfoundry.operations.DefaultCloudFoundryOperations;
+import org.yaml.snakeyaml.Yaml;
+import picocli.CommandLine;
+
+import java.io.IOException;
+
+/**
+ * This class realizes the functionality that is needed for the create commands.
+ */
+@CommandLine.Command(name = "create", header = "%n@|green Create-Controller|@", subcommands = {
+    CreateController.CreateServiceCommand.class,
+    CreateController.AssignSpaceDeveloperCommand.class,
+    CreateController.CreateApplicationCommand.class })
+public class CreateController implements Runnable {
+    @Override
+    public void run() {
+        // this code is executed if the user runs the create command without specifying
+        // any sub-command
+    }
+
+    @CommandLine.Command(name = "space-developer", description = "Assign users as space developers")
+    static class AssignSpaceDeveloperCommand implements Runnable {
+        @CommandLine.Mixin
+        LoginCommandOptions loginOptions;
+        @CommandLine.Mixin
+        CreateControllerCommandOptions commandOptions;
+
+        @Override
+        public void run() {
+            String yamlFileContent;
```

Comment:
Please don't be afraid of adding a few blank lines in your code. Now it's so tightly packed, one can hardly tell which lines are related logically.